integration_test.go

```diff
@@ -22,6 +24,39 @@ func TestSSH(t *testing.T) {
 	if !*integration {
 		t.Skip()
 	}
+
+	for _, tt := range sshTests {
+		got, err := getHostname(tt.hostname)
+		if err != nil {
+			t.Fatal(err)
+		}
+		want := fmt.Sprintf("%s\n", tt.hostname)
+		if got != want {
+			t.Fatalf("want = %#v, got = %#v", want, got)
+		}
+	}
+
```

Comment:
If you fix https://github.com/k1LoW/sshc/pull/6#discussion_r269094690 , could you fix tests too ?

Reply:
I fixed test in https://github.com/k1LoW/sshc/pull/6/commits/c2ed0d3621dda5b21af9cf3c18f736a81c62c6b5 .